Assets/ScaleMonk Ads/Source/Android/AdsBindingInterstitialListener.java

```diff
@@ -10,19 +10,19 @@
     @Override
     public void onInterstitialView(@Nullable String tag) {
         Log.d(AdsBinding.TAG, "Completed interstitial display at location " + tag);
-        UnityPlayer.UnitySendMessage("ScaleMonkAdsMonoBehavior", "CompletedInterstitialDisplay", tag);
+        UnityPlayer.UnitySendMessage("AdsMonoBehaviour", "CompletedInterstitialDisplay", tag);
     }
 
     @Override
     public void onInterstitialClick(@Nullable String tag) {
         Log.d(AdsBinding.TAG, "Clicked interstitial at location " + tag);
-        UnityPlayer.UnitySendMessage("ScaleMonkAdsMonoBehavior", "ClickedInterstitial", tag);
```

Comment:
Why are we changing the game object name?

Reply:
It needs to match the game object that is created [here](https://github.com/scalemonk/mediation-sdk-unity/blob/10ff1f03354ee6e24d4442a8eaee708cab411588/Assets/ScaleMonk%20Ads/Source/ScaleMonkAdsMonoBehavior.cs#L33), iOS was matching but Android wasn't, and thus it gets no messages.